src/app/category.ts

```diff
@@ -0,0 +1,4 @@
+export interface Category {
+    id: number
+    name: string
```

Comment:
add slug parameter

Reply:
Slug will be intergrated will using the datas from US-Interfaces